### DISCUSSION TITLE
limit reports allowed by user role

### DISCUSSION THREAD

**Discussion by sgtrwe** [sheltermanager/asm3#1597] [Q&A / Help] [answered]
I have a role that limits the user to view only, but I only want them to be able to run reports from one section of the reports menu, is there any way to do this ?  If there is, I haven't found it.

Thanks

**Comment by bobintetley** [comment, marked as answer]
You can assign roles to your reports under Settings->Reports. Once a report has roles assigned, then only the users with one of those roles can run it or see it in the menu.

**Reply by sgtrwe** [reply]
Thanks that was it.
